### PR TITLE
[REFACTOR] notification listener

### PR DESCRIPTION
**Pull request by iariasleon**

### CODE REVIEW

File: test/acceptance/behave/README.md [hunk 1]
```diff
@@ -194,6 +194,10 @@ file in the same directory that contains the “steps” directory.
     * MONGO_DATABASE: mongo database (sth by default).
     * MONGO_RETRIES: number of retries for data verification.
     * MONGO_DELAY_TO_RETRY: time in seconds to delay in each retry.
+    
+ Properties used by listeners
+- listeners
+    *  LISTENER_NOTIF: listener host used to notfications
```
Comment (fgalan) [inline test/acceptance/behave/README.md]: used for notifications

or

used to notify

File: test/acceptance/behave/README.md [hunk 2]
```diff
@@ -389,6 +393,7 @@ The Context Broker must start with "DEBUG" level in "CB_EXTRA_OPS" field into pr
           | metadatas_type      | "alarm"                             |
           | metadatas_value     | "default"                           |
         ```             
+  - if "attr_value" has "timestamp in last minutes" as value is generated a timestamp with N last minutes of current timestamp
```
Comment (fgalan) [inline test/acceptance/behave/README.md]: if ..., then a timestamp with N last minutes of current timestamp is generated

Comment (iariasleon) [inline test/acceptance/behave/README.md]: Fixed in  	1d5e950

File: test/acceptance/behave/README.md (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/behave/README.md]: Fixed in 1d5e950

**Comment by fgalan** [comment]
LGTM

**Pull request merged**